app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -665,6 +665,11 @@ def build_toolbar_hide_button(id)
       when "condition_remove"
         return true if x_active_tree == :condition_tree || !role_allows(:feature => "condition_delete")
       end
+    when "CustomButtonSet"
+      case id
+      when "ab_group_edit", "ab_group_delete", "ab_button_new"
+        return !role_allows(:feature => "catalogitem_admin") if x_active_tree == :sandt_tree
```

Comment:
@jzigmund i think we should check for the specific features add/edit of composite & atomic catalog items, user might not have access to remove catalog items but they can add or edit them so if they can add or edit catalog items they should be able to add buttons and button groups.